supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesInitializationActorSpec.scala

```diff
@@ -0,0 +1,70 @@
+package cromwell.backend.impl.jes
+
+import akka.actor.ActorSystem
+import akka.testkit.{ImplicitSender, TestDuration, TestKit}
+import com.typesafe.config.ConfigFactory
+import cromwell.backend.BackendWorkflowInitializationActor.{InitializationFailed, InitializationSuccess, Initialize}
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor}
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
+import spray.json.{JsObject, JsValue}
+import wdl4s.values.WdlValue
+import wdl4s.{Call, NamespaceWithWorkflow, WdlSource}
+
+import scala.concurrent.duration._
+
+class JesInitializationActorSpec extends TestKit(ActorSystem("JesInitializationActorSpec"))
+  with WordSpecLike with Matchers with BeforeAndAfterAll with ImplicitSender {
+  val Timeout = 1.second.dilated
+
+  val HelloWorld =
+    """
+      |task hello {
+      |  String addressee = "you"
+      |  command {
+      |    echo "Hello ${addressee}!"
+      |  }
+      |  output {
+      |    String salutation = read_string(stdout())
+      |  }
+      |
+      |  RUNTIME
+      |}
+      |
+      |workflow hello {
+      |  call hello
+      |}
+    """.stripMargin
+
+  val defaultBackendConfig = new BackendConfigurationDescriptor("jes-config", ConfigFactory.load())
+
+  private def buildWorkflowDescriptor(wdl: WdlSource,
+                                      inputs: Map[String, WdlValue] = Map.empty,
+                                      options: WorkflowOptions = WorkflowOptions(JsObject(Map.empty[String, JsValue])),
+                                      runtime: String = "") = {
+    new BackendWorkflowDescriptor(
+      WorkflowId.randomId(),
+      NamespaceWithWorkflow.load(wdl.replaceAll("RUNTIME", runtime)),
+      inputs,
+      options
+    )
+  }
+
+  private def getJesBackend(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], conf: BackendConfigurationDescriptor) = {
+    system.actorOf(JesInitializationActor.props(workflowDescriptor, calls, conf))
+  }
+
+  "JesInitializationActor" should {
+    "return InitializationSuccess when there are no runtime attributes defined." in {
```

Comment:
Not in this PR perhaps, but could you add a ticket for adding significantly more tests here! :)